kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h

```diff
@@ -229,6 +218,25 @@ class ResidualBasedIncrementalAitkenStaticScheme : public ResidualBasedIncrement
         mOldOmega = Omega;
     }
 
+    static Parameters StaticGetDefaultParameters()
```

Comment:
why is this needed, why cannot you use the defaut  method?

Reply:
Nope, it is used in the constructor, does not compile